### PR TITLE
modules

### PR DESCRIPTION
**Pull request by festinahuruglica**
I created just acf-tools.php and new directory in app acf inside acf.php, the other files (changes from others) when i created the branch comes together

### CODE REVIEW

File: acf-tools.php [hunk 1]
```diff
@@ -0,0 +1,165 @@
+<?php
+
+add_action( 'acf/include_fields', function() {
```
Comment (andiemini7) [inline acf-tools.php]: Remove this file

File: app/acf/acf.php [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+			),
+			'min' => '',
+			'max' => '',
+			'button_label' => 'Add Row',
+		),
+	),
+	'location' => array(
+		array(
+			array(
+				'param' => 'post_type',
```
Comment (andiemini7) [inline app/acf/acf.php:101]: Generate the location automatically for all pages/cpt

File: app/acf/acf.php [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+	'instruction_placement' => 'label',
+	'hide_on_screen' => '',
+	'active' => true,
+	'description' => '',
+	'show_in_rest' => 0,
+) );
+} );
+
+add_action( 'init', function() {
+	register_post_type( 'modul', array(
```
Comment (andiemini7) [inline app/acf/acf.php]: Remove this custom post type

File: template-modules.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
+<?php
+/* Template Name: Modules Page */
+
+get_header(); ?>
+
+<div id="primary" class="content-area">
+    <main id="main" class="site-main" role="main">
```
Comment (andiemini7) [inline template-modules.php]: Remove this file.

File: page.php [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<?php get_header(); ?>
+
+<div id="primary" class="content-area">
+    <main id="main" class="site-main" role="main">
+        <?php
+        while (have_posts()) : the_post();
+            the_content();
+
```
Comment (andiemini7) [inline page.php]: We need to change this using another method

**Review by andiemini7 — CHANGES_REQUESTED** [review]

**Comment by festinahuruglica** [comment]
Adding the location for custom post types, removing the extra files.

**Pull request merged**